src/client/utils/sortLists.js

```diff
@@ -0,0 +1,47 @@
+import * as moment from 'moment';
+/**
+ *
+ * @param {*} criteria - Criteria argument has to match databse object property. eg. const user = { name: 'Fred'} -> criteria = 'name'
+ */
+const dateFormat = 'YYYY-MM-DDTHH:mm:ss.SSSSZ';
+
+export const sortList = (listName, criteria, order) =>
+  order
```

Comment:
this could be simplified (without code copying) by moving the 'order' check inside the is statements

Reply:
Check if I did it the correct way. If no give me some tip how to make it better